project/plugins.sbt

```diff
@@ -9,3 +10,5 @@ addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.1")
 addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.8")
 
 addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "3.8.1")
+
+addSbtPlugin("org.scalameta" % "sbt-mdoc" % "2.2.8" )
```

Comment:
@raquo btw my 2.13 changes for mdoc just got published in new version **2.2.9**, so if I didn't break it you should be able to switch back to 2.13 as the default build version